app/src/viewmodels/home/MaximizeScreenViewModel.tsx

```diff
@@ -56,6 +57,23 @@ export function useMaximizeScreenViewModel(
     });
   }, [challenge, firestoreCtrl, currentUserId]);
 
+  // Fetch user data
+  useEffect(() => {
+    const fetchUser = async () => {
+      try {
+        if (
+          user.image_id?.startsWith("http") ||
+          user.image_id?.startsWith("https://")
```

Comment:
Same question here. What is the reason of this divide?

Reply:
Same as previous